app/models/measure.rb

```diff
@@ -393,7 +393,10 @@ def verbose_duty_expression
   end
 
   def prettify_generated_duty_expression(duty_expression)
-    duty_expression.sub(/\s%/, '%').sub(/\s\s/, ' ')
+    duty_expression.sub!(/\s\s/, ' ')
+    duty_expression.sub!(/\s%/, '%') if duty_expression.scan(/\d\s%/).present?
+    duty_expression.sub!(/\/\s[a-zA-Z]/, &:downcase) if duty_expression.scan(/\/\s[a-zA-Z]/).present?
+    duty_expression
```

Comment:
In general, mutating a passed in variable can often lead to later bugs when another developer consumes a method without expecting it to mutate the data, particularly in this case where it both mutates the variable passed in, then returns the mutated form anyway.

Ruby in general is moving away from mutating strings with the push towards frozen strings literals (as seen in other languages)

I'd suggest either appending the method name with an exclamation mark to show its going to mutate data (eg difference between `'#sub` and `#sub!`. Or alternatively with the suggestions above, you can drop the `if` conditions and just chain the substitutions